legato-api/src/main/java/org/opendaylight/legato/util/LegatoUtils.java

```diff
@@ -0,0 +1,386 @@
+				.build();
+	}
+
+	public static final CreateConnectivityServiceInput1 buildCreateConServiceAugmentation(String maxFrameSize) {
+		CreateConnectivityServiceInput1 createConServiceAugmentation = new CreateConnectivityServiceInput1Builder()
+				.setNrpCarrierEthConnectivityResource(buildNrpCarrierEthConnectivityResource(maxFrameSize)).build();
+		return createConServiceAugmentation;
+	}
+
+	public static final UpdateConnectivityServiceInput1 buildUpdateConServiceAugmentation(String maxFrameSize) {
+		UpdateConnectivityServiceInput1 updateConServiceAugmentation = new UpdateConnectivityServiceInput1Builder()
+				.setNrpCarrierEthConnectivityResource(buildNrpCarrierEthConnectivityResource(maxFrameSize)).build();
+		return updateConServiceAugmentation;
+	}
+
+	public static final NrpCarrierEthConnectivityEndPointResource buildNrpCarrierEthConnectivityEndPointResource(
+			String vlanId) {
+
+		NrpCarrierEthConnectivityEndPointResourceBuilder nrpCarrierEthConnectivityEndPointResourceBuilder = new NrpCarrierEthConnectivityEndPointResourceBuilder();
+
+		CeVlanIdListAndUntagBuilder ceVlanIdListAndUntagBuilder = new CeVlanIdListAndUntagBuilder();
+		List<VlanId> vlanList = new ArrayList<VlanId>();
+		VlanIdBuilder vlanIdBuilder = new VlanIdBuilder().setVlanId(new PositiveInteger(Long.parseLong(vlanId)));
+		vlanList.add(vlanIdBuilder.build());
+
+		ceVlanIdListAndUntagBuilder.setVlanId(vlanList);
+
+		nrpCarrierEthConnectivityEndPointResourceBuilder.setCeVlanIdListAndUntag(ceVlanIdListAndUntagBuilder.build());
+		nrpCarrierEthConnectivityEndPointResourceBuilder.setIngressBwpFlow(buildIngressBwFlow());
+
+		return nrpCarrierEthConnectivityEndPointResourceBuilder.build();
+
+	}
+
+	public static final IngressBwpFlow buildIngressBwFlow() {
+		IngressBwpFlowBuilder ingressBwpFlowBuilder = new IngressBwpFlowBuilder()
+				.setRank(new PositiveInteger(LegatoConstants.LONG_VAL))
+				.setCir(new NaturalNumber(LegatoConstants.LONG_VAL))
+				.setCirMax(new NaturalNumber(LegatoConstants.LONG_VAL))
+				.setCbs(new NaturalNumber(LegatoConstants.LONG_VAL)).setEir(new NaturalNumber(LegatoConstants.LONG_VAL))
+				.setEirMax(new NaturalNumber(LegatoConstants.LONG_VAL))
+				.setEbs(new NaturalNumber(LegatoConstants.LONG_VAL)).setCouplingFlag(true)
+				.setColorMode(ColorMode.COLORAWARE).setTokenRequestOffset(new NaturalNumber(LegatoConstants.LONG_VAL));
+
+		return ingressBwpFlowBuilder.build();
+	}
+
+	public static final NrpCarrierEthConnectivityResource buildNrpCarrierEthConnectivityResource(String maxFrameSize) {
+		NrpCarrierEthConnectivityResourceBuilder nrpCarrierEthConnectivityResourceBuilder = new NrpCarrierEthConnectivityResourceBuilder();
+		return nrpCarrierEthConnectivityResourceBuilder
+				.setMaxFrameSize(new PositiveInteger(Long.parseLong(maxFrameSize))).build();
+	}
+
+	public static final CreateConnectivityServiceInput buildCreateConnectivityServiceInput(Map<String, Object> evcMap,
```

Comment:
using ``Map<String,Object>`` for evc representation might be hard to maintain in the long run
* you need to know type of value under each key
* you need to use ton of casts
* you can learn that there is a problem only at runtime
* nobody will stop another person contributing to this code to make it in a wrong way

You can consider ``class EvcParamsDto`` when instead of key values you have typed attributes:

e.g. 
instead ``(List<String) EVC_UNI_LIST`` you have ``List<String> EvcParamsDto.getEvcUniList()``
instead ``evcMap.get(LC.EVC_MAX_FRAME).toString() you have ``long EvcParamsDto.getMaxFrameSize()``